### DISCUSSION TITLE
Conditional auto profile for videos with specific fps

### DISCUSSION THREAD

**Discussion by noobworth** [mpv-player/mpv#14236] [Q&A] [answered]
How can I setup an auto profile to apply for videos with a specific framerate?
I tried doing this but it didn't work:
```
[ntsc-video]
profile-cond=p["container_fps"] == 23.976
profile-restore=copy
d3d11-sync-interval=2
display-fps-override=47.952
video-sync=display-resample

[pal-video]
profile-cond=p["container_fps"] == 25
profile-restore=copy
d3d11-sync-interval=2
display-fps-override=50
video-sync=display-resample
```

**Comment by guidocella** [comment, marked as answer]
`profile-cond=math.abs(container_fps - 23.976) < 0.001`
`profile-cond=container_fps == 25`

**Reply by noobworth** [reply]
That worked perfectly, thank you!

**Comment by yeezylife** [comment]
I tried
‘display-fps-override=’
’video-sync=display-resample‘  on a g-sync monitor and still got mistimed frames growing every second and high v-sync jitters. there are a lot of discussion and still don't know how to properly use ‘video-sync=display-resample’ on a g-sync/adaptive-sync monitor

**Reply by guidocella** [reply]
`display-resample` is incompatible with adaptive sync.

**Reply by yeezylife** [reply]
> `display-resample` is incompatible with adaptive sync.

now i use `video-sync=audio`(default) and `display-fps-override=my display's refresh rate-3`(to trigger g-sync)...seems to work. atleast no mistimed or droped frames anymore
